downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc

```diff
@@ -17,27 +17,13 @@ You can install {Navigator} on Red Hat Enterprise Linux (RHEL) from an RPM.
 
 .Procedure
 
-. Attach the {PlatformName} SKU.
-+
-----
-$ subscription-manager attach --pool=<sku-pool-id>
-----
+. Install {Navigator} using the following:
 
-. Enable the repository for RHEL 8.
-+
+[options="nowrap" subs="+quotes"]
 ----
-$ sudo subscription-manager repos --enable ansible-automation-platform-2.1-for-rhel-8-x86_64-rpms
+sudo dnf install --enablerepo=ansible-automation-platform-2.2-for-rhel-8-x86_64-rpms ansible-navigator
 ----
```

Comment:
I think it's a good thing to remove the dollar sign at the command prompt but is this consistent with the rest of the guide?